doc/sphinx/source/utils/index.rst

```diff
@@ -0,0 +1,9 @@
+Utilities
+=========
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Utilities
+
+   utils
```

Comment:
Is it necessary to push the existing utilities down a level? 
![image](https://github.com/ESMValGroup/ESMValTool/assets/16133375/5ac71147-2240-4958-a76e-76d177e3f5db)

I would would have expected RTW to be included in the main list of utilities.

Reply:
That makes sense as an ultimate goal. Is there another short-term workaround that would keep them all at the same level? E.g. adding a very short RTW section to `utils.rst`, e.g. containing a single sentence and link to `RTW/index`. There could still then be a follow-up issue to implement your suggested long-term approach.

Reply:
I can do this, but the `Recipe Test Workflow (RTW)` link still shows in the sidebar, just on it's own.

"All documents in the [source directory](https://www.sphinx-doc.org/en/master/glossary.html#term-source-directory) (or subdirectories) must occur in some toctree directive" (see [Sphinx documentation: Directives](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html)), so I have the RTW files in a toctree directive, but this automatically populates the sidebar, and I can't figure out how to tell Sphinx to not do that without removing the sidebar entirely 😭 Maybe it would be easier for me to move the utilities now? 🤔

Reply:
> I can do this, but the Recipe Test Workflow (RTW) link still shows in the sidebar, just on it's own.

I think that's still better that having Utilities > Utilities in the hierarchy?

Reply:
Great - that looks good to me. Thanks!

I'll approve now. One final suggestion, which you are welcome to ignore... is it worth duplicating the following sentence into the top-level utils index, so that someone can see at a glance what RTW is?
"The Recipe Test Workflow (RTW) is a workflow that is used to regularly run recipes so issues can be discovered during the development process sooner rather than later."

Reply:
Apologies, I didn't see this before I merged! I'll pop this in when I work on the final PR 👍

Reply:
See #3685 😊